### PR TITLE
RC Reorg - Accounts and settings

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft of updating content for Account Setting and User Profile content

### CODE REVIEW

**Comment by lanceleonard** [comment]
Staged previews are available:
- [Account Settings](https://docs.redis.com/staging/jira-doc-1333/rc/accounts/account-settings/) page
- [User profile page](https://docs.redis.com/staging/jira-doc-1333/rc/accounts/user-profile/)
- [Cloud account settings](https://docs.redis.com/staging/jira-doc-1333/rc/cloud-integrations/aws-cloud-accounts/) page.

File: content/rc/accounts/account-settings.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+| _Date created_ | Date the user's Redis Cloud account was created, which may differ from the organization account creation date |
+| <nobr>_Owner email address_</nobr> | Email address used to create the owner's account |
+| _Account number_ | Internal ID of the owner's account |
+| _Last updated_ | Date of the last administrative change to the owner's account, typically reflect access changes or other administrative updates | 
+
+You cannot change the email address associated with a Redis Cloud account.  Instead, create a new account with the updated email address, assign it as an administrator to the organization account, and then use the new account to delete the account with the invalid email address.
+
+## Account address settings
+
+The **Account address** section shows the address associate with the current Redis Cloud account and the current time zone.
```
Comment (kaitlynmichael) [inline content/rc/accounts/account-settings.md:42]:
```suggestion
The **Account address** section shows the address associated with the current Redis Cloud account and the current time zone. 
```

File: content/rc/accounts/account-settings.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+To update the time zone, select the desired time zone from the **Time zone** drop-down.
+
+In addition, this section may include fields unique to your location.  For example, certain regions require tax IDs or other regulatory details.
+
+When updating details in this section, pay particular attention to prompts and error messages.
+
+## Security settings
+
+The security section lets you:
```
Comment (kaitlynmichael) [inline content/rc/accounts/account-settings.md:52]: Is this supposed to be repeated twice?

**Review by kaitlynmichael — APPROVED** [review]
Two small suggestions, otherwise looks good.

File: content/rc/accounts/account-settings.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Integration settings
+
+The **Integrations settings** section includes settings that help you manage the integration of your Redis Cloud account with your underlying cloud provider.  For example, subscriptions hosted on AWS can [map their accounts]({{<relref "/rc/administration/aws-zone-mapping">}}) to specific availability zones.
+
+If this section doesn't appear on the **Account Settings** screen, it generally means that there aren't any settings for current account to manage.
+
+## Save or discard changes
+
+Few account settings can be changed, however, you can update a few details, such as **Time Zone**, **MFA enforcement**, and AWS availability zone mapping.
```
Comment (rrelledge) [inline content/rc/accounts/account-settings.md:68]: Maybe reword this slightly, break up the sentence, or change the comma before "however" to a semicolon?
```suggestion
Although most account settings cannot be changed, you can update a few details, such as **Time Zone**, **MFA enforcement**, and AWS availability zone mapping.
```

File: content/rc/accounts/user-profile.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+{{<image filename="images/rc/account-selector-single-account.png" alt="Use the Profile control to manage your user account profile and to switch between Redis Cloud accounts." width="300px">}}{{< /image >}}
+
+When you open the Profile control, you can:
+
+- Review and manage your user account profile
+
+- Sign out from the admin console
+
+- Switch between Redis Cloud subscriptions administered by your user account.
```
Comment (rrelledge) [inline content/rc/accounts/user-profile.md:26]:
```suggestion
- Review and manage your user account profile.

- Sign out from the admin console.

- Switch between Redis Cloud subscriptions administered by your user account.
```

File: content/rc/accounts/user-profile.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+To review your user profile settings, select **User profile** from the Profile control.  This displays the **User Profile** screen:
+
+{{<image filename="images/rc/user-profile-settings.png" alt="The User Profile screen lets you manage selected settings associated with your user account." width="75%">}}{{< /image >}}
+
+This screen contains up to three sections, including:
+
+- The *User details* section includes basic information about your account, including _Name_, _Job title_, _Email_, and the date the account was created.  _Name_ and _Job title_ can be edited; the other settings are read-only.
+
+- The *Password* lets you change the password for for accounts created and managed by Redis Cloud.
```
Comment (rrelledge) [inline content/rc/accounts/user-profile.md:38]:
```suggestion
- The *Password* section lets you change the password for accounts created and managed by Redis Cloud. 
```

File: content/rc/accounts/user-profile.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+{{<image filename="images/rc/user-profile-settings.png" alt="The User Profile screen lets you manage selected settings associated with your user account." width="75%">}}{{< /image >}}
+
+This screen contains up to three sections, including:
+
+- The *User details* section includes basic information about your account, including _Name_, _Job title_, _Email_, and the date the account was created.  _Name_ and _Job title_ can be edited; the other settings are read-only.
+
+- The *Password* lets you change the password for for accounts created and managed by Redis Cloud. 
+
+    If you're using single sign-on, you cannot change the password using the *User Profile* screen.  Such accounts are managed by an identity provider (IdP).  For help changing (or revoering) the passwords on these accounts, consult your identity provider docs.
```
Comment (rrelledge) [inline content/rc/accounts/user-profile.md:40]:
```suggestion
    If you're using single sign-on, you cannot change the password using the *User Profile* screen.  Such accounts are managed by an identity provider (IdP).  For help changing (or recovering) the passwords for these accounts, consult your identity provider docs.
```

File: content/rc/accounts/user-profile.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+- The **Multi-factor authentication (MFA)** section lets you manage MFA settings for the current user account.  
+
+    When you activate a mobile device, you can use SMS MFA to as a second authentication factor.
```
Comment (rrelledge) [inline content/rc/accounts/user-profile.md:44]:
```suggestion
    When you activate a mobile device, you can use SMS MFA as a second authentication factor.
```

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md [hunk 8]
```diff
@@ -20,46 +20,102 @@ Customers with existing Amazon Web Services (AWS) accounts can provision their F
...
+|Setting|Description|
+|-------|-----------|
+| _Account name_ | A descriptive name for your cloud account settings |
+| _AWS access key_ | The AWS access key for the programmatic user created to support your cloud account settings |
+| _AWS secret key_ | The AWS secret key for the programmatic user created to support your cloud account settings |
+| <nobr>_AWS console username_</nobr> | The username for the AWS console |
+| _AWS console password_ | The password for AWS console access |
 
-    - Select the account that you want to edit and then select **Edit**.
+Use the **Update account** button so save your changes.
```
Comment (rrelledge) [inline content/rc/cloud-integrations/aws-cloud-accounts/_index.md:107]:
```suggestion
Use the **Update account** button to save your changes.
```

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md [hunk 9]
```diff
@@ -20,46 +20,102 @@ Customers with existing Amazon Web Services (AWS) accounts can provision their F
...
-- Manually suspend or stop (terminate) provisioned resources</br></br>
+- Manually suspend or stop (terminate) provisioned resources</br>
 {{</note>}}
 
-## Add or edit a cloud account
+## View cloud account settings
 
 To create or edit a cloud account in Redis Enterprise Cloud:
 
 1. Sign into the [admin console](https://app.redislabs.com/) and then select the target subscription.
```
Comment (rrelledge) [inline content/rc/cloud-integrations/aws-cloud-accounts/_index.md:40]:
```suggestion
1. Sign in to the [admin console](https://app.redislabs.com/) and then select the target subscription.
```

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md [hunk 10]
```diff
@@ -20,46 +20,102 @@ Customers with existing Amazon Web Services (AWS) accounts can provision their F
...
+| _Account name_ | A descriptive name for your cloud account settings |
+| _AWS&nbsp;access&nbsp;key_ | The AWS access key for the programmatic user created to support your cloud account settings |
+| _AWS&nbsp;secret&nbsp;key_ | The AWS secret key for the programmatic user created to support your cloud account settings |
+| _IAM role name_ | The name of the AWS console role with access to the AWS console |
+
+Use the **Add account** button to save your cloud account details.
+
+{{<image filename="images/rc/button-cloud-account-add.png" alt="Use the Add account button to save the details of your new cloud account." width="140px">}}{{< /image >}}
+
+Be sure to create the resources before adding the cloud account to your subscription, as they're used to verify access to the cloud account.  The details can be saved oly after access is verified.
```
Comment (rrelledge) [inline content/rc/cloud-integrations/aws-cloud-accounts/_index.md:75]:
```suggestion
Be sure to create the resources before adding the cloud account to your subscription, as they're used to verify access to the cloud account.  The details can be saved only after access is verified.
```

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md [hunk 11]
```diff
@@ -20,46 +20,102 @@ Customers with existing Amazon Web Services (AWS) accounts can provision their F
...
+
+For help, see [Create IAM resources]({{<relref "/rc/cloud-integrations/aws-cloud-accounts/iam-resources">}}).
+
+## Edit cloud account details
+
+To update the details of a cloud account associated with your Redis Cloud subscription, select the cloud account from the **Cloud account** tab and then select the **Edit** button.
+
+{{<image filename="images/rc/icon-cloud-account-edit.png" alt="Use the Edit button to update cloud account details." width="36px">}}{{< /image >}}
+
+This displays the **Edit cloud account dialog**:
```
Comment (rrelledge) [inline content/rc/cloud-integrations/aws-cloud-accounts/_index.md:95]:
```suggestion
This displays the **Edit cloud account** dialog:
```

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md [hunk 12]
```diff
@@ -20,46 +20,102 @@ Customers with existing Amazon Web Services (AWS) accounts can provision their F
...
-1. Select **Save**.
+To remove a cloud account from your Redis cloud subscription, select the cloud account from the **Cloud account** tab and then select the **Delete** button.
 
-Use the **Delete** button to remove a cloud account from your subscription.
+{{<image filename="images/rc/icon-cloud-account-delete.png" alt="Use the Delete button to remove cloud account details." width="36px">}}{{< /image >}}
 
 ## Dedicated IAM resources
 
-We recommend creating dedicated identity and access management (IAM) resources to manage the infrastructure of your subscriptions.
+We strongly recommend using dedicated identity and access management (IAM) resources to manage your ASW cloud accounts.  These resources should not be shared with any other task, account, or process.
```
Comment (rrelledge) [inline content/rc/cloud-integrations/aws-cloud-accounts/_index.md:119]:
```suggestion
We strongly recommend using dedicated identity and access management (IAM) resources to manage your AWS cloud accounts.  These resources should not be shared with any other task, account, or process.
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I left suggestions for some formatting and typo fixes.

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md (hunk 11 above)
Comment (lanceleonard) [reply content/rc/cloud-integrations/aws-cloud-accounts/_index.md:95]: Fixed.

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md (hunk 10 above)
Comment (lanceleonard) [reply content/rc/cloud-integrations/aws-cloud-accounts/_index.md:75]: Fixed.

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md (hunk 9 above)
Comment (lanceleonard) [reply content/rc/cloud-integrations/aws-cloud-accounts/_index.md:40]: Fixed.

File: content/rc/cloud-integrations/aws-cloud-accounts/_index.md (hunk 8 above)
Comment (lanceleonard) [reply content/rc/cloud-integrations/aws-cloud-accounts/_index.md:107]: Fixed.

File: content/rc/accounts/user-profile.md (hunk 7 above)
Comment (lanceleonard) [reply content/rc/accounts/user-profile.md:44]: Fixed.

File: content/rc/accounts/user-profile.md (hunk 6 above)
Comment (lanceleonard) [reply content/rc/accounts/user-profile.md:40]: Fixed.

File: content/rc/accounts/user-profile.md (hunk 4 above)
Comment (lanceleonard) [reply content/rc/accounts/user-profile.md:26]: Fixed.

File: content/rc/accounts/account-settings.md (hunk 3 above)
Comment (lanceleonard) [reply content/rc/accounts/account-settings.md:68]: Fixed.

File: content/rc/accounts/account-settings.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+Use the **Save changes** button to save changes or the **Discard changes** to revert them.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/rc/accounts/account-settings.md:70]: Fixed.

File: content/rc/accounts/account-settings.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+
+The security section lets you:
+
+The security section lets you:
+
+- Manage [multi-factor authentication]({{<relref "/rc/security/multi-factor-authentication">}}) (MFA) for your Redis Cloud account 
+
+- Download [certificate authority]({{<relref "/rc/security/database-security/tls-ssl#certificates">}}) (CA) certificates for the certificates associated with your Redis Cloud account
+
+## Integration settings
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/rc/accounts/account-settings.md:60]: It should, except when it looks funny or raises additional questions.  Fixed by making sure the text matches the UX.

File: content/rc/accounts/account-settings.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+The **Account Info** section provides basic details about your account, including:
+
+| Setting | Description |
+|---------|-------------|
+| _Owner name_ | Person associated with the Redis Cloud account |
+| _Account name_ | Organization associated with the Redis Cloud account | 
+| _Date created_ | Date the user's Redis Cloud account was created, which may differ from the organization account creation date |
+| <nobr>_Owner email address_</nobr> | Email address used to create the owner's account |
+| _Account number_ | Internal ID of the owner's account |
+| _Last updated_ | Date of the last administrative change to the owner's account, typically reflect access changes or other administrative updates |
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/rc/accounts/account-settings.md:36]: Fixed

File: content/rc/accounts/account-settings.md [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+
+This displays the **Account Settings** screen:
+
+{{<image filename="images/rc/account-settings-account-tab.png" alt="Use the Account tab of the Account Settings screen to review and update settings associated with your Redis Cloud account." width="75%">}}{{< /image >}}
+
+The available tabs depend on your subscription type and may include:
+
+- The **Account** tab displays basic information associated with your account, including general info, address details, time zone setting, security settings, and provider integration details.
+
+- The **Cloud Accounts** tab is displayed for Flexible (or Annual) subscriptions hosted on Amazon Web Services (AWS).  To learn more, see [Manage AWS cloud accounts]({{<relref "rc/cloud-integrations/aws-cloud-accounts/">}})
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/rc/accounts/account-settings.md:23]: Fixed.

File: content/rc/accounts/account-settings.md (hunk 2 above)
Comment (lanceleonard) [reply content/rc/accounts/account-settings.md:52]: Nope.  Nope.  Fixed.  Fixed.

File: content/rc/accounts/account-settings.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/accounts/account-settings.md:42]: Fixed.
